cudax/include/cuda/experimental/__async/basic_sender.cuh

```diff
@@ -82,23 +82,23 @@ struct basic_receiver
   __state<_Data, _Rcvr>& __state_;
 
   template <class... _Args>
-  _CCCL_HOST_DEVICE _CUDAX_ALWAYS_INLINE void set_value(_Args&&... __args) noexcept
+  _CCCL_HOST_DEVICE _CUDAX_HIDDEN_INLINE void set_value(_Args&&... __args) noexcept
   {
     __rcvr_t::set_value(__state_.__data_, __state_.__receiver_, (_Args&&) __args...);
   }
 
   template <class _Error>
-  _CCCL_HOST_DEVICE _CUDAX_ALWAYS_INLINE void set_error(_Error&& __error) noexcept
+  _CCCL_HOST_DEVICE _CUDAX_HIDDEN_INLINE void set_error(_Error&& __error) noexcept
   {
     __rcvr_t::set_error(__state_.__data_, __state_.__receiver_, (_Error&&) __error);
   }
 
-  _CCCL_HOST_DEVICE _CUDAX_ALWAYS_INLINE void set_stopped() noexcept
+  _CCCL_HOST_DEVICE _CUDAX_HIDDEN_INLINE void set_stopped() noexcept
   {
     __rcvr_t::set_stopped(__state_.__data_, __state_.__receiver_);
   }
 
-  _CCCL_HOST_DEVICE _CUDAX_ALWAYS_INLINE decltype(auto) get_env() const noexcept
+  _CCCL_HOST_DEVICE _CUDAX_HIDDEN_INLINE decltype(auto) get_env() const noexcept
```

Comment:
I believe those should all rather be
```suggestion
  _LIBCUDACXX_HIDE_FROM_ABI decltype(auto) get_env() const noexcept
```

I dont see any benefit here from adding forceinline

Reply:
the primary motivation for using `_CUDAX_HIDDEN_INLINE` here is because i want this fn to disappear from debugging. i don't want to have to step through it, and i don't want it cluttering up stack-traces. it's just plumbing.

i have used this technique in stdexec to improve the debugging experience. see: https://x.com/ericniebler/status/1719031544881549467